src/StatusQ/Popups/StatusModal.qml

```diff
@@ -11,13 +11,19 @@ import "statusModal" as Spares
 QC.Popup {
     id: statusModal
 
+    property alias advancedHeader: advancedHeader.item
+    property alias advancedFooter: advancedFooter.item
+    property alias advancedHeaderComponent: advancedHeader.sourceComponent
+    property alias advancedFooterComponent: advancedFooter.sourceComponent
```

Comment:
Can you add documentation for these while you're at it?
Also, can we get an example of what an advanced footer/header is in the page component?
To me it looks exactly like the normal header/footer from an API perspective so it's unclear why this is needed

Reply:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=2798%3A147680 this is where you can see new design for wallet and carl mentioned this would be updated in StatusModal.

The way that this is different is the freedom to add a Header and Footer as any component. Earlier we could only assign some buttons to the header and footer

Reply:
Iv added an example to the demoApp, and added some documentation for Modal now.